src/useForm.tsx

```diff
@@ -50,9 +50,10 @@ export function useForm<T extends Record<string, any>>(
     return !hasValue
   }, [state])
 
-  const getValueCallback = useCallback(() => getValue(state), [state])
-
-  return { getValue: getValueCallback, validate, fields, isEmpty, reset }
+  const getValue = useCallback(() => extractValuesFromFieldsState(state), [
```

Comment:
I renamed `getValue` => `extractValuesFromFieldsState` which is imo a bit clearer as to what this function really does